calendar-bundle/contao/templates/events/event_teaser.html5

```diff
@@ -34,7 +34,7 @@
   </div>
 
   <?php if ($this->hasDetails): ?>
-    <p class="more"><a href="<?= $this->href ?>" title="<?= $this->readMore ?>"<?= $this->target ?>><?= $this->more ?><span class="invisible"> <?= $this->link ?></span></a></p>
+    <p class="more"><a href="<?= $this->href ?>" aria-label="<?= $this->readMore ?>"<?= $this->target ?>><?= $this->more ?></a></p>
```

Comment:
These changes are wrong. The `title` attribute needs to stay so the browser tooltip shows `Read the article: My article` in addition to the link text, which is always just `Read more …`. Screen readers ignore the `title` attribute and shall read `Read more … My article`, that's why we must not remove the `<span class="invisible">`.